db/c_test.c

```diff
@@ -93,6 +93,11 @@ static void CheckEqual(const char* expected, const char* v, size_t n) {
   }
 }
 
+static unsigned char CheckOptionsEqual(const rocksdb_options_t* left,
+                                       const rocksdb_options_t* right) {
+  return memcmp(left, right, 16) == 0;
```

Comment:
this still feels like too much magic here... at least you want to add a comment to explain where 16 comes from. otherwise people won't understand this without reading the comments in this PR.

Reply:
That totally makes sense, I have added the comment.